guide/private-key-management/cloud-backup.md

```diff
@@ -86,6 +88,12 @@ To see what a user experience with this scheme could look like, see the [daily s
 - [Casa](https://keys.casa)
```

Comment:
I bumped the size of the subheadings up 1 in the `Best practice` section in my external signers PR, looks a little more legible imo.

Reply:
Done